blackbox/docs/sql/statements/copy-from.rst

```diff
@@ -296,6 +296,45 @@ This option specifies the format of the input file. Available formats are
 ``csv`` or ``json``. If a format is not specified and the format cannot be
 guessed from the file extension, the file will be processed as JSON.
 
+.. _return_summary:
+
+``RETURN SUMMARY``
+------------------
+
+By using the optional ``RETURN SUMMARY`` clause, a summary result set will be
+returned describing possible failures and success per involved data node and
+resource.
+
+::
+
+    [ RETURN SUMMARY ]
+
+.. rubric:: Schema
+
++--------------------------------+------------------------------------------------+---------------+
+| Column Name                    | Description                                    |  Return Type  |
++================================+================================================+===============+
+| ``node``                       | Information about the node that is processing  | ``OBJECT``    |
+|                                | the URI resource.                              |               |
++--------------------------------+------------------------------------------------+---------------+
+| ``node['id']``                 | The id of the node.                            | ``STRING``    |
++--------------------------------+------------------------------------------------+---------------+
+| ``node['name']``               | The name of the node.                          | ``STRING``    |
++--------------------------------+------------------------------------------------+---------------+
+| ``uri``                        | The URI the node is processing.                | ``STRING``    |
++--------------------------------+------------------------------------------------+---------------+
+| ``error_count``                | The total number of records which failed.      | ``LONG``      |
++--------------------------------+------------------------------------------------+---------------+
+| ``success_count``              | The total number of records which succeeded.   | ``LONG``      |
++--------------------------------+------------------------------------------------+---------------+
+| ``errors``                     | Contains detailed information about possible   | ``OBJECT``    |
+|                                | errors.                                        |               |
++--------------------------------+------------------------------------------------+---------------+
+| ``errors[ERROR_MSG]``          | Contains information about a type of an error. | ``OBJECT``    |
```

Comment:
Why is the ERROR_MSG capitalized and not in single quotes?

Reply:
because it is a placeholder, it will be e.g. "Duplicate Key"